lightbluetent/api.py

```diff
@@ -206,10 +243,21 @@ def is_meeting_running(self, params):
 
     def request(self, call, params):
         url = self.build_url(call, params)
-        res = requests.get(url, timeout=(0.5, 10))
+
+        try:
+            res = requests.get(url, timeout=(0.5, 10))
+
+        except requests.exceptions.ReadTimeout:
+            current_app.logger.info(f"Timeout timed out! Requests.exceptions.ReadTimeout when making API call { call }")
+            return None
+
         xml = xmltodict.parse(res.text)
 
-        return xml["response"]
+        if "response" not in xml:
+            current_app.logger.info(f"Malformed response from server: { xml }")
+            return None
+        else:
+            return xml["response"]
 
 
     def generate_checksum(self, call, query):
```

Comment:
I understand this wasn't originally written by you, but can we just make all the BBB methods static? Having to instantiate just to use the methods is weird. See: https://julien.danjou.info/guide-python-static-class-abstract-methods/